qcodes/instrument_drivers/Gentec/Gentec_Maestro.py

```diff
@@ -52,16 +52,17 @@ def zero_offset_get_parser(ans):
         # print connect message
         self.connect_message()
 
-    # get methods
-    def get_idn(self):
-        return {'vendor': 'Gentec', 'model': self.model, 'firmware': self.firmware_version}
+        # get methods
```

Comment:
These changes do not look like they belong here ?

Reply:
The build for the docs were failing because of this. So I did the changes here. There is a problem then again. I will check it from the master branch.

Reply:
Sorry for just jumping in, but the added indentation is wrong. The methods `get_idn`, `clear_zero_offset`, `set_zero_offset`, and `_query_version` should be where they were, i.e. as bound methods of the class, not as methods inside the scope of `__init__`.

Reply:
Yes I corrected it now.